lib/camp.js

```diff
@@ -768,18 +783,22 @@ function pathLikeUnit(serverMethod, statusCode) {
           for (var key in matched) {
             req.data[key] = matched[key];
           }
-          if (callbacks[cbindex] !== undefined) {
-            callbacks[cbindex](req, res);
+          var cb = callbacks[cbindex];
+          var method = req.method.toLowerCase();
```

Comment:
Can we avoid that lowercase computation when this is not a method-specific filter?